### PR TITLE
Moved content to right of sidebar

### PR DESCRIPTION
**Pull request by TylerWasniowski**

### CODE REVIEW

File: packages/frontend/src/components/routes.jsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// @flow
+
+export const HomeRoute: String = '/';
```
Comment (vikr01) [inline packages/frontend/src/components/routes.jsx]: If this file is not using `react`, its file extension should be `.js` for consistency.

Comment (vikr01) [inline packages/frontend/src/components/routes.jsx]: You should use the lowercase `string` type with flow.

Comment (TylerWasniowski) [reply packages/frontend/src/components/routes.jsx]: Okay, thanks

Comment (TylerWasniowski) [reply packages/frontend/src/components/routes.jsx]: Okay, what is the difference?

Comment (vikr01) [reply packages/frontend/src/components/routes.jsx]: https://flow.org/en/docs/types/primitives/

Comment (TylerWasniowski) [reply packages/frontend/src/components/routes.jsx]: Thanks!

**Pull request merged**